src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -133,6 +133,65 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void getCopy_shouldCreateDeepCopy() {
+        String accountGoogleId = "johnDoe";
+
+        AccountAttributes account = AccountAttributes.builder()
+                .withGoogleId(accountGoogleId)
+                .withName("John")
+                .withEmail("johnDoeEmail")
+                .withInstitute("johnson")
+                .withIsInstructor(true)
+                .build();
+        account.isInstructor = true;
+
+        account.studentProfile = StudentProfileAttributes.builder()
```

Comment:
Same here. You can also try `.withDefaultStudentProfileAttributes(accountGoogleId)` instead of writting so much test data.